assets/js/components/buy-checkout.component.js

```diff
@@ -95,6 +93,12 @@ function BuyCheckoutController ($scope, $timeout, $q, currency, Wallet, MyWallet
     this.handleBuy({ quote }).finally($scope.resetFields).finally($scope.free);
   };
 
+  $scope.setLimits = (limit) => {
+    $scope.min = currency.convertToSatoshi(0.01, $scope.dollars);
+    $scope.max = currency.convertToSatoshi(limit, $scope.dollars);
+  };
+
+  $scope.$watch('$ctrl.buyLimit', (limit) => limit && $scope.setLimits(limit));
```

Comment:
I think found an interesting bug here, when the limit is 0 this won't set the limits. Will fix.